ophyd/_dummy_shim.py

```diff
@@ -23,3 +23,7 @@ def get_pv(*args, **kwargs):
 
 def release_pvs(*args, **kwargs):
     raise NotImplementedError
+
+
+def get_dispatcher():
+    raise NotImplementedError
```

Comment:
This will break the dummy backend as we unconditionally call this in `Signal.__init__`.

Reply:
Looks like we need some dummy backend tests...